include/xlnt/worksheet/worksheet.hpp

```diff
@@ -433,9 +433,11 @@ class XLNT_API worksheet
     column_t highest_column_or_props() const;
 
     /// <summary>
-    /// Returns a range_reference pointing to the full range of non-empty cells in the worksheet.
+    /// Returns a range_reference pointing to the full range of cells in the worksheet.
+    /// If skip_null is true (default), empty cells (For example if the first row or column is empty)
```

Comment:
maybe `skip_empty`?

Reply:
sure, I was using the same naming convention as the caller (rows() and columns())

Reply:
nevermind I see you're copying the name